assertion_error: test_alphabetic_sorting

I'm running Python 3.6.0 for macos 10.13.2

test_isort.py:1660 (test_alphabetic_sorting)
def test_alphabetic_sorting():
        """Test to ensure isort correctly handles single line imports"""
        test_input = ("import unittest\n"
                      "\n"
                      "import ABC\n"
                      "import Zope\n"
                      "from django.contrib.gis.geos import GEOSException\n"
                      "from plone.app.testing import getRoles\n"
                      "from plone.app.testing import ManageRoles\n"
                      "from plone.app.testing import setRoles\n"
                      "from Products.CMFPlone import utils\n"
                      )
        options = {'force_single_line': True,
                   'force_alphabetical_sort_within_sections': True, }
    
        output = SortImports(file_contents=test_input, **options).output
>       assert output == test_input
E       assert 'import unitt...mport utils\n' == 'import unitte...mport utils\n'
E           import unittest
E         - 
E         - from django.contrib.gis.geos import GEOSException
E           
E           import ABC
E           import Zope
E         + from django.contrib.gis.geos import GEOSException
E           from plone.app.testing import getRoles
E           from plone.app.testing import ManageRoles
E           from plone.app.testing import setRoles
E           from Products.CMFPlone import utils

test_isort.py:1677: AssertionError

--- a/test_isort.py
+++ b/test_isort.py
@@ -1673,7 +1673,7 @@ def test_alphabetic_sorting():
     options = {'force_single_line': True,
                'force_alphabetical_sort_within_sections': True, }
 
-    output = SortImports(file_contents=test_input, **options).output
+    output = SortImports(file_contents=test_input, known_first_party=['django'], **options).output
     assert output == test_input
 
     test_input = ("# -*- coding: utf-8 -*-\n"

